src/Response.php

```diff
@@ -107,7 +107,7 @@ public function write($data)
         return $flushed;
     }
 
-    public function end($data = null)
+    public function end($data = null, $keepAlive = false)
```

Comment:
Can you give some example code on how you expect this to be used?

It's my understanding that the `Response` object should behave like a `WritableStreamInterface`, i.e. end()ing this only terminates its "virtual stream" and should not have any effect on the underlying socket connection. IMO it should be up to the server to determine *when* the close the underlying connection.

Any info on how other HTTP libs handle this?